harmonica/coordinates.py

```diff
@@ -0,0 +1,47 @@
+"""
+Geographic coordinate conversion.
+"""
+import numpy as np
+
+from . import get_ellipsoid
+
+
+def geodetic_to_geocentric(latitude, height):
+    """
+    Convert from geodetic to geocentric coordinates.
```

Comment:
```suggestion
    Convert from geodetic to geocentric spherical coordinates.
```

Reply:
Actually, this makes me think if we might want a different name for the functions. This might be confused with geocentric cartesian coordinates.

Reply:
On the other hand, we also don't want a huge function name. Maybe just `geocentric_spherical`? Or maybe we leave it as is and trust that people will read the docs. But we might also need a geocentric Cartesian converter later on.

Reply:
Every time I was writing the function name I needed to rethink if i wasn't confusing `geodetic_to_geocentric` with `geocentric_to_geodetic`, so sounds good idea to change its name.
Replacing geocentric with spherical makes it more rememberable and prevents the confusion with Cartesian geocentric coordinates, so I would propose `geodetic_to_spherical` in order to illustrate that the function will convert coordinates (I think that `geocentric_spherical` doesn't imply a coordinate conversion, could be confusing when reading/writing data processing code).

And keeping this naming rule, a future geodetic to Cartesian conversion could be named `geodetic_to_cartesian`.

Reply:
:+1: sounds good